trackpy/plots.py

```diff
@@ -257,6 +257,56 @@ def pairwise(iterable):
     return ax
 
 
+def annotate3d(centroids, image, **kwargs):
+    """
+    An extension of annotate that distinguishes between 2D, 2D multichannel,
+    3D, and 3D multichannel images, just as in pims.frame.__repr_html_. See
+    annotate for parameters. Returns either (for 2D) an axis object or (for 3D)
+    a scrollable stack.
+    """
+    import matplotlib.pyplot as plt
+    from pims.display import to_rgb, scrollable_stack
+
+    if kwargs.get('ax') is None:
```

Comment:
Note that the other functions in this module have a decorator @make_axes (defined above) that take care of this `ax` magic. You will need to put `ax=None` into the signature between `image` and `**kwargs`.